### DISCUSSION TITLE
OnlineSpeakerDiarization' object has no attribute 'from_source'

### DISCUSSION THREAD

**Discussion by amitli1** [juanmc2005/diart#140] [General]
I'm trying to run the example:

![image](https://user-images.githubusercontent.com/128060257/229537464-ba9c4332-c73d-4ea5-a707-f0fdfcbcda27.png)

```
from diart.sources   import MicrophoneAudioSource
from diart.sinks     import RealTimePlot
from diart.operators import buffer_output

duration, step, latency, sample_rate = 5, 0.5, 0.5, 16000
mic = MicrophoneAudioSource(sample_rate)
OnlineSpeakerDiarization().from_source(mic).pipe(buffer_ouput(duration, step, latency, sample_rate)).subscribe(RealTimePlot(duration, latency))

```

And I'm getting the error:
`AttributeError: 'OnlineSpeakerDiarization' object has no attribute 'from_source'`

I'm using the following versions:
```
diart==0.6.0
pyannote.audio==2.1.1
```

How can I run this example ?

**Comment by juanmc2005** [comment]
Hi @amitli1, as mentioned in #132, the example from the image is a bit outdated. The API has had some changes since then.

Could you try with the example [here](https://github.com/juanmc2005/StreamingSpeakerDiarization#from-python)?

**Reply by amitli1** [reply]
It works, 
Thanks
